Addons/PolyQuilt/subtools/subtool_brush_relax.py

```diff
@@ -90,7 +90,7 @@ def CollectVerts( self , context , coord ) :
 
         select_stack.push()
         select_stack.select_mode(True,False,False)
-        bpy.ops.view3d.select_circle( x = coord.x , y = coord.y , radius = radius , wait_for_input=False, mode='SET' )
+        bpy.ops.view3d.select_circle( x = int(coord.x), y = int(coord.y) , radius = int(radius) , wait_for_input=False, mode='SET' )
```

Comment:
bpy.ops.view3d.select_circle( x = int(round(coord.x)) , y = int(round(coord.y)) , radius = int(round(radius)) , wait_for_input=False, mode='SET' ) 

in 3.5 im doing this code that fix the relax tool error,